src/JestExt/core.ts

```diff
@@ -176,7 +185,8 @@ export class JestExt {
             messaging.systemErrorMessage(
               event.error,
               messaging.showTroubleshootingAction,
-              this.setupWizardAction('cmdLine')
+              this.setupWizardAction('cmdLine'),
+              this.setupIgnoreAction(event.workspaceFolder)
```

Comment:
I think we should only add the additional action for multi-root workspaces, something like
```
if (vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length > 1) {
...
}
```

Reply:
🤔 on second thought i do also have single folder workspaces that dont work with jest where disabling it with one click might also be nice

Reply:
for single folder workspace, I think the current activation check should be sufficient; and they can simply disable the extension via vscode if it's truly not applicable. However, multi-root workspace is different:
1. we did not perform the same activation check for each multiroot workspace folder
2. users can't simply disable the extension for a certain workspace folder via vscode, thus the introduction of the option `jest.disabledWorkspaceFolders` for more granular control.